pkg/controller/command/didexchange/command.go

```diff
@@ -8,15 +8,16 @@ package didexchange
 
 import (
 	"encoding/json"
-	"errors"
 	"fmt"
 	"io"
 
 	"github.com/hyperledger/aries-framework-go/pkg/client/didexchange"
 	"github.com/hyperledger/aries-framework-go/pkg/common/log"
 	"github.com/hyperledger/aries-framework-go/pkg/controller/command"
 	"github.com/hyperledger/aries-framework-go/pkg/controller/internal/cmdutil"
+	"github.com/hyperledger/aries-framework-go/pkg/controller/webnotifier"
```

Comment:
It feels odd that we need to import a package with the prefix of `web`. Do we need a followup to fix the naming (or split up the generic notifications from the `web` ones)?